unison-core/src/Unison/Util/Tritraversable.hs

```diff
@@ -0,0 +1,19 @@
+{-# LANGUAGE DefaultSignatures #-}
+
+module Unison.Util.Tritraversable where
```

Comment:
This is how you know we're real functional programmers 🙃

Reply:
Right now in the `codebase-sqlite` package, there's `trimap` and `quadmap` for Patch and Namespace, respectively. I'm kind of leaning "delete it entirely" and just put `trimap/`tritraverse`/`quadmap`/`quadtraverse` as needed next to the corresponding types, given that I suspect we only have one of each?

Reply:
Otherwise yeah, it could be a new util package?

Reply:
Hm, yeah; I'm not sure about this, either. I think the code that uses this abstraction would be easier to understand if it just did three normal traversals.